### ISSUE TITLE
Update sourced-ce

### ISSUE THREAD

**Issue by dpordomingo** [src-d/sourced-ce#44]
We could have an automated way to upgrade `sourced` once it has been locally installed.

To update `sourced`, the steps to be (manually) done, are docmmented by
https://docs.sourced.tech/community-edition/learn-more/faq-troubleshooting#how-can-i-update-my-version-of-source-d-ce

(from @smacker [slack thread](https://src-d.slack.com/archives/C7TB5NEDN/p1560248262114700))

**Comment by smacker** [comment]
It was simplified a bit by replacing step 5 and 6 by `sourced restart`

**Comment by se7entyse7en** [comment]
Linking the DD: https://docs.google.com/document/d/1lQaygo8iTibPhYSuNtkIV0l41tYNioPK6I0eKP2IKO4/edit#
\cc @rpau

**Comment by se7entyse7en** [comment]
Just to avoid ideas vanish linking here links from engine:
- https://github.com/src-d/engine-deprecated/issues/308
- https://github.com/src-d/engine-deprecated/pull/323#issuecomment-472757370

**Comment by se7entyse7en** [comment]
update: one round of review from @src-d/product already done, some comments need to be addressed on our-side, once approved this is ready to go.
